ci-operator/step-registry/cucushift/installer/check/aws/custom-tag/cucushift-installer-check-aws-custom-tag-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: cucushift-installer-check-aws-custom-tag
+  from: upi-installer
```

Comment:
If the step does not depend on openshift-instal version, I would suggest to use a tag name of upi-installer, so that the step does not depend on up-installer config in ci yaml.